src/main/scala/mesosphere/marathon/api/v2/json/Formats.scala

```diff
@@ -71,7 +71,8 @@ trait Formats
       "ports" -> task.getPortsList.asScala,
       "startedAt" -> (if (task.getStartedAt != 0) Timestamp(task.getStartedAt) else JsNull),
       "stagedAt" -> (if (task.getStagedAt != 0) Timestamp(task.getStagedAt) else JsNull),
-      "version" -> task.getVersion
+      "version" -> task.getVersion,
+      "slaveId" -> task.getSlaveId.getValue
```

Comment:
We cannot rely on the slaveId being present. Does this code correctly deal with that case?